.github/workflows/deploy-to-production.yml

```diff
@@ -3,6 +3,9 @@ name: Deploy to production
 on:
   push:
     branches: [ master ]
+  schedule:
+    # Run every 3 hours
+    - cron: '0 */3 * * *'
```

Comment:
I'm wondering if we should even reduce this to run every hour? That way the update time will almost be completely accurate. It would also sync well with the github token rate limit reset time.

Reply:
every hour makes sense!